crates/provider/src/ext/erc4337.rs

```diff
@@ -0,0 +1,191 @@
+    async fn eth_get_user_operation_receipt(
+        &self,
+        user_op_hash: Bytes,
+    ) -> TransportResult<UserOperationReceipt>;
+
+    /// Estimates the gas for a [`UserOperation`].
+    async fn eth_estimate_user_operation_gas(
+        &self,
+        user_op: UserOperation,
+        entry_point: Address,
+    ) -> TransportResult<UserOperationGasEstimation>;
+}
+
+#[cfg_attr(target_arch = "wasm32", async_trait::async_trait(?Send))]
+#[cfg_attr(not(target_arch = "wasm32"), async_trait::async_trait)]
+impl<N, T, P> Erc4337Api<N, T> for P
+where
+    N: Network,
+    T: Transport + Clone,
+    P: Provider<T, N>,
+{
+    async fn eth_send_user_operation(
+        &self,
+        user_op: UserOperation,
+        entry_point: Address,
+    ) -> TransportResult<SendUserOperationResponse> {
+        self.client().request("eth_sendUserOperation", (user_op, entry_point)).await
+    }
+
+    async fn eth_supported_entry_points(&self) -> TransportResult<Vec<Address>> {
+        self.client().request("eth_supportedEntryPoints", ()).await
+    }
+
+    async fn eth_get_user_operation_receipt(
+        &self,
+        user_op_hash: Bytes,
+    ) -> TransportResult<UserOperationReceipt> {
+        self.client().request("eth_getUserOperationReceipt", (user_op_hash,)).await
+    }
+
+    async fn eth_estimate_user_operation_gas(
+        &self,
+        user_op: UserOperation,
+        entry_point: Address,
+    ) -> TransportResult<UserOperationGasEstimation> {
+        self.client().request("eth_estimateUserOperationGas", (user_op, entry_point)).await
+    }
+}
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use crate::ProviderBuilder;
+    use alloy_node_bindings::Geth;
+    use alloy_primitives::{Address, Bytes, U256};
+
+    #[tokio::test]
+    async fn test_eth_send_user_operation() {
+        let temp_dir = tempfile::TempDir::with_prefix("geth-test-").unwrap();
+        let geth = Geth::new().disable_discovery().data_dir(temp_dir.path()).spawn();
+        let provider = ProviderBuilder::new().on_http(geth.endpoint_url());
+
+        let user_op = UserOperation {
+            sender: Address::random(),
+            nonce: U256::from(0),
+            factory: Address::random(),
+            factory_data: Bytes::default(),
+            call_data: Bytes::default(),
+            call_gas_limit: U256::from(1000000),
+            verification_gas_limit: U256::from(1000000),
+            pre_verification_gas: U256::from(1000000),
+            max_fee_per_gas: U256::from(1000000000),
+            max_priority_fee_per_gas: U256::from(1000000000),
+            paymaster: Address::random(),
+            paymaster_verification_gas_limit: U256::from(1000000),
+            paymaster_post_op_gas_limit: U256::from(1000000),
+            paymaster_data: Bytes::default(),
+            signature: Bytes::default(),
+        };
+
+        let entry_point: Address = "0x5FF137D4b0FDCD49DcA30c7CF57E578a026d2789".parse().unwrap();
```

Comment:
EntryPoint 0.7 is address 0x0000000071727De22E5E9d8BAf0edAc6f37da032

Reply:
![image](https://github.com/user-attachments/assets/4ba2d672-16f8-4365-a58b-bd6cba21dabf)
Yes!